plugins/pantheon-files-ctags/plugin.vala

```diff
@@ -318,14 +295,23 @@ public class Marlin.Plugins.CTags : Marlin.Plugins.Base {
     }
 
     private async void set_color (GLib.List<unowned GOF.File> files, int n) throws IOError {
+        if (files == null) {
```

Comment:
Not sure why this check is needed here. Instead I would expect callers of this method to always pass a valid list of `files`.